### PR TITLE
CCP-1803: Updated for compatability with PWA 6.8.0

### PR DESCRIPTION
**Pull request by dqdquint**
AddToCartButton component from PWA is no longer extendable. Copied implementation instead of passing a onClick prop because of additional methods added.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx [hunk 1]
```diff
@@ -1,22 +1,81 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showCheckmark: null,
     };
   }
 
+  /**
+ * Component did update lifecycle hook.
```
Comment (aaron-martin) [inline frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx]: indention problem

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx [hunk 2]
```diff
@@ -1,22 +1,81 @@
...
+    if (this.props.addedQuantity > prevProps.addedQuantity) {
+      // Trigger the button animation when the quantity was increased.
+      setTimeout(() => {
+        this.onQuantityIncreased();
+      }, 250);
+    }
+  }
+
+  /**
+* Called when the added quantity was increased. It triggers the button animation.
```
Comment (aaron-martin) [inline frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx]: indention problem

File: frontend/portals/ProductItemFavoritesButton/index.jsx [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+import CTAButtons from './components/CTAButtons';
 
 /**
  * Renders CTA buttons in product-item.favorites-button portal
  * @param {Object} props props
  * @returns {JSX}
  */
-const ProductItemFavoritesButton = props => <CTAButtons {...props} />;
+const ProductItemFavoritesButton = (props) => {
+  return (<CTAButtons {...props} />);
```
Comment (aaron-martin) [inline frontend/portals/ProductItemFavoritesButton/index.jsx]: what is the advantage of using a return? Why not 
```
import React from 'react';
import CTAButtons from './components/CTAButtons';

/**
 * Renders CTA buttons in product-item.favorites-button portal
 * @param {Object} props props
 * @returns {JSX}
 */
const ProductItemFavoritesButton = props => (
  <CTAButtons {...props} />
);

export default ProductItemFavoritesButton;
```

**Review by aaron-martin — CHANGES_REQUESTED** [review]

File: frontend/portals/ProductItemFavoritesButton/index.jsx (hunk 3 above)
Comment (dqdquint) [reply frontend/portals/ProductItemFavoritesButton/index.jsx]: I think I may have made this change in testing and didn't revert to a non bodied return

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx (hunk 2 above)
Comment (dqdquint) [reply frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx]: fixed

File: frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx (hunk 1 above)
Comment (dqdquint) [reply frontend/portals/ProductItemFavoritesButton/components/CTAButtons/components/AddToCartPicker/components/AddToCartPickerButton/index.jsx]: fixed

File: frontend/portals/ProductItemFavoritesButton/index.jsx (hunk 3 above)
Comment (dqdquint) [reply frontend/portals/ProductItemFavoritesButton/index.jsx]: fixed

**Review by aaron-martin — APPROVED** [review]

**Pull request merged**